docs/en/docs/sql-manual/sql-reference/Show-Statements/SHOW-CATALOG-RECYCLE-BIN.md

```diff
@@ -0,0 +1,62 @@
+---
+{
+    "title": "SHOW-CATALOG-RECYCLE-BIN",
+    "language": "en"
+}
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## SHOW-CATALOG-RECYCLE-BIN
+
+### Name
+
+SHOW CATALOG RECYCLE BIN
+
+### Description
+
+This statement is used to display the dropped meta informations that can be recovered
+
+语法：
+
+```sql
+SHOW CATALOG RECYCLE BIN [ WHERE NAME [ = "name" | LIKE "name_matcher"] ]
+```
```

Comment:
Please give a result example to explain the columns in result.

Reply:
And also need to update `docs/sidebars.json`

Reply:
done